go/flowctl/cmd-combine.go

```diff
@@ -0,0 +1,156 @@
+package main
```

Comment:
I'm not really convinced this has long-term utility in `flowctl`. Maybe we keep this code around, but I don't htink we want it exposed to users.

Reply:
Agreed. I made it a hidden subcommand for now, and we can fully remove it later if it's still not useful for testing.